docs/user_stories_devops_engineer.md

```
@@ -0,0 +1,217 @@
+
+### 1. The "Platform Paved Road"
+
+Priya created a standard GitHub Action that all teams use:
+
+```yaml
+# .github/workflows/asp-review.yml (company standard)
+name: ASP Code Review
+
+on:
+  pull_request:
+    types: [opened, synchronize]
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)